25_exceptions_lab/02.py

```diff
@@ -1,7 +1,15 @@
-"""
-Write a program that takes a file name
-and prints line count for the file
-
-Alert the user politely if there was any problem opening the file
-"""
-
+import sys
+import os
+
+src = sys.argv[1]
+
+line_count = 0
+
+if ( not os.path.isfile(src)):

```

Comment:
No need to check if `src` is a file:

1. Open could fail for a million other reasons, so it's not that productive anyways
2. If everything's ok, you just wasted time checking something that's true

A better (more pythonic) way is to use `except` after `with open`, handling exceptions if they happen